### ISSUE TITLE
Give option to bind hotkey for switching window mode

### ISSUE THREAD

**Issue by makedir** [moonlight-stream/moonlight-chrome#417]
The default ESC bind is stupid, because it is overwritten by most games. You just can switch to window mode while the connecting... window happens. If you are in the game, it goesnt work anymore, and closes the game if pressed ESC (most games close by this).

- give option for hotkey this
- give option to NOT start default into fullscreen mode

**Comment by Jorys-Paulin** [comment]
@makedir 
- What keybinding should be used then?
- It should be possible

**Comment by makedir** [comment]
@Jorys-Paulin Anything other than ESC and something which is not used anywhere or with high chance not used. ESC closes most games, or opens the game menu. Best would be alt+enter, because it is the default binding for switching window/fullscreen, at last under Windows.

**Comment by Jorys-Paulin** [comment]
@makedir On Windows 10, I tried `Atl+Enter` with no success, however `F11` worked fine. It is also the shortcut for ChromeOS, platform for which moonlight-chrome was initially started

**Comment by makedir** [comment]
@Jorys-Paulin No. alt+enter is the correct normal Window binding to switch between windowed and fullscreen applications, mostly since Windows 95. 

https://support.microsoft.com/en-us/help/81823/alt-enter-switches-between-window-and-full-screen

Please dont change it to any single binding like F11. Something like ant+F11 maybe, but better give option to make it changeable for the user.

**Comment by Jorys-Paulin** [comment]
@makedir Well windows explorer does nothing when `Alt+Enter`, but does fullscreen with `F11`, but I get your point.

**Comment by makedir** [comment]
@Jorys-Paulin Well, I would be fine with a change to F11 for now, until a user change would be implemented. Anything other than ESC like F11 would be ok.

**Comment by Jorys-Paulin** [comment]
@makedir I made a quick & dirty build of your feature request, you can only change fullscreen or windowed at startup for now.
[moonlight-chrome.zip](https://github.com/moonlight-stream/moonlight-chrome/files/1763536/moonlight-chrome.zip)

**Comment by makedir** [comment]
@Jorys-Paulin but that wouldnt help much. I need to toggle fullscreen/window a lot like every few minutes. Can you please make another dirty build of changing ESC to F11, that would help a lot. Or wont that work after the game started? Is the keybinding intercepted on the client first?

**Comment by Jorys-Paulin** [comment]
@makedir I don't really know since the cursor (and so the keyboard) is captured by the C++ module, I don't know if I'll be able to capture it using javascript.

**Comment by makedir** [comment]
@Jorys-Paulin  but where does the esc binding come from? didnt you implement it? same with when clicking on the maximize button, it goes into fullscreen instead of maximizing the window.

**Comment by Jorys-Paulin** [comment]
@makedir It's all handled by chrome

**Comment by makedir** [comment]
@Jorys-Paulin If I click on maximize on another app like Google Keep it does not go into fullscreen, so that is a behavior of your app.

**Comment by Jorys-Paulin** [comment]
@makedir My fault, it is made so [here](https://github.com/moonlight-stream/moonlight-chrome/blob/master/static/js/index.js#L23)

**Comment by Jorys-Paulin** [comment]
@makedir I think I resolved your issue with this pull request (#435), let me know what you think!

**Comment by makedir** [comment]
@Jorys-Paulin where? I don't see any builds but one from march 28th.

**Comment by Jorys-Paulin** [comment]
@makedir It has to be accepted (marked merged) to be in the official builds

**Comment by makedir** [comment]
@Jorys-Paulin and how would I test it then? I don't see any test build download links on the site.

**Comment by Jorys-Paulin** [comment]
@makedir Let me make you a build, I'll be back in a minute

**Comment by Jorys-Paulin** [comment]
@makedir Sorry, it is taking sooo long to upload...

**Comment by Jorys-Paulin** [comment]
@makedir Here's a build. Please note that I wasn't able to add a hotkey :/
[window-mode.zip](https://github.com/moonlight-stream/moonlight-chrome/files/1998389/window-mode.zip)

**Comment by Jorys-Paulin** [comment]
@makedir So, is it a build you'll accept? What should I change?

**Comment by makedir** [comment]
@elegos I said that before up there. This is redundant, because I want to switch in game back and forth from Window to full-screen and this doesnt work so far.
